bundles/org.openhab.binding.tr064/README.md

```diff
@@ -35,48 +37,97 @@ For security reasons it is highly recommended to set both, username and password
 
 ### `fritzbox`
 
-All additional parameters for `fritzbox` devices (i.e. except those that are shared with `generic`) are advanced parameters.
+The `fritzbox` devices can give additional informations in dedicated channels, controlled  by additional parameters, w.r.t. to `generic` devices.
+These parameters are advanced parameters, i.e. they appear in the Main User Interface only if the `Show Advanced` option is selected.
+If the parameters are specified, the corresponding channels will be added to the device. 
 
-One or more TAM (telephone answering machines) are supported by most devices.
+One or more TAM (telephone answering machines) are supported by most fritzbox devices (namely the ones with DECT support).
```

Comment:
This creates the impression that DECT needs to be enabled to make the TAM work. That is not the case, I have only IP phones and the TAM works fine.

Reply:
ok. i removed  the mention of DECT